packages/common-i18n/src/locales/en/search.json

```diff
@@ -64,5 +66,14 @@
   },
   "goToVenue": "Show page",
   "openStreetMapContributors": "contributors",
-  "mapAccessibilityStatement": "Map. Currently map information is only accessible visually."
+  "mapAccessibilityStatement": "Map. Currently map information is only accessible visually.",
+  "accessibilityShortcoming": {
+    "noAccessibilityProfile": "No accessibility shortcoming",
+    "hearing_aid": "Hearing aid",
+    "reduced_mobility": "Reduced mobility",
+    "rollator": "Rollator",
+    "stroller": "Stroller",
+    "visually_impaired": "Visually impaired",
+    "wheelchair": "Wheelchair"
+  }
```

Comment:
IMHO it'd be good to use the same translations here as in https://palvelukartta.hel.fi/en/ as we are using their exact translations for in the Finnish version already.